Makefile

```diff
@@ -99,6 +99,7 @@ test-backend: install
 .PHONY: test-frontend
 test-frontend: install
 	yarn build
```

Comment:
We might not need this build step. I think we only included this before there were any tests to catch the most basic frontend issues.

Reply:
Will `$ yarn test` still display tying issues?

Reply:
Nope; gotta have both.